rrrspec-client/lib/rrrspec/client/support.rb

```diff
@@ -40,7 +40,26 @@ def start_taskset(conf, rsync_name)
         return taskset
       end
 
-      def show_result(taskset, verbose=false)
+      def show_result(taskset, verbose=false, show_errors=false)
+        if show_errors
+          puts "Errors:"
+          taskset.tasks.each do |task|
+            if task.status == "failed"
+              trial = task.trials.first
+              stdout = trial.stdout
+              if stdout
+                puts "STDOUT:"
```

Comment:
Stack trace shows failed spec file, but separate line with failed spec file name should be more better.